tests/integration/test_blocking.py

```diff
@@ -0,0 +1,304 @@
+from _pytest.python_api import raises
+
+from redis.exceptions import ConnectionError
+from .sandbox import RawConnection
+
+
+def test_brpop(cluster):
```

Comment:
maybe you can have some tests for bad timeout values, does redis-suite run those tests? Still, I think we can have a one test here.

Reply:
redis-suite does those tests, so that's why I skipped it.  but I guess.

Reply:
done, copied what redis tests do.